gripper/firmware/motor_hardware_g.c

```diff
@@ -0,0 +1,377 @@
+#include "motor_hardware.h"
+
+#include "common/firmware/errors.h"
+#include "stm32g4xx_hal.h"
+
+DAC_HandleTypeDef hdac1;
+
+TIM_HandleTypeDef htim1;
+TIM_HandleTypeDef htim2;
+TIM_HandleTypeDef htim3;
+TIM_HandleTypeDef htim4;
+
+TIM_OC_InitTypeDef htim1_sConfigOC = {0};
+TIM_OC_InitTypeDef htim3_sConfigOC = {0};
+
+
+uint32_t round_closest(uint32_t dividend, uint32_t divisor) {
```

Comment:
This and `calc_prescaler` either belong in `motor_hardware_shared.c` or they can be static functions.